### PR TITLE
Movie page

### PR DESCRIPTION
**Pull request by HaniaNassalska**

### CODE REVIEW

File: src/Features/Movies/SingleMovie/Core/Credits/index.js [hunk 1]
```diff
@@ -14,6 +15,7 @@ const Credits = ({ credits }) => {
   return (
     <CreditWrapper>
       {credits && credits.map((credit, id) => (
+        <StyledLink to={`/profile/${credit.id}`}>
```
Comment (KubaTuta) [inline src/Features/Movies/SingleMovie/Core/Credits/index.js:18]: this way you would have doubled keys, please correct like line below

Comment (KubaTuta) [inline src/Features/Movies/SingleMovie/Core/Credits/index.js]: this way you would have doubled keys, please correct like line below

File: src/Features/Movies/SingleMovie/Core/Credits/index.js [hunk 2]
```diff
@@ -14,6 +15,7 @@ const Credits = ({ credits }) => {
...
         <CreditPeopleWrapper key={id}>
```
Comment (KubaTuta) [inline src/Features/Movies/SingleMovie/Core/Credits/index.js]: here key is unnecessery now

**Review by KubaTuta — APPROVED** [review]
minor key correction but rest is great, you could consider shortening cast and crew tiles tho, maybe some expanding (...) sign?

**Review by Mateusszzp — APPROVED** [review]
It's obvious that you pay great attention to detail. Everything is fine with mine

**Pull request merged**